core/src/main/scala/org/apache/spark/TestUtils.scala

```diff
@@ -159,6 +159,16 @@ private[spark] object TestUtils {
     createCompiledClass(className, destDir, sourceFile, classpathUrls)
   }
 
+  def createResource(
```

Comment:
This isn't worth a utility method you use once. it's a one-liner everywhere else, a call to `Files.write`